### DISCUSSION TITLE
Add a link to the org or the repo on the new issues/projects feed

### DISCUSSION THREAD

**Discussion by jeffrafter** [community/community#15688] [Projects and Issues]
Hi FRIENDS YOU ARE AMAZING 💖 

What would you think about adding a link to the org or the repo from the projects, right here:

![image](https://user-images.githubusercontent.com/4064/165580589-1fadb33b-a5fd-44db-aa70-6cfc7c5b41a6.png)

THANK YOU

**Comment by Elbossmusic** [comment]
Let’s work together
